custom_components/sector/coordinator.py

```diff
@@ -360,3 +396,15 @@ async def _async_update_data(self):
         except Exception as error:
             _LOGGER.exception("Failed to update data")
             raise UpdateFailed(f"Failed to update data: {error}") from error
+
+    def _filter_duplicate_logs(self, logs):
```

Comment:
If we always just grab the latest event from the entity, there should be no need to filter anything.

Reply:
Replaced with history lookup of the event-log to only process newer events than the last event processed.